app/models/user.rb

```diff
@@ -35,6 +35,21 @@ def update_with_password(params)
     super
   end
 
+  # @see Devise::Confirmable
+  # send_confirmation_instructions
+  def send_confirmation_instructions
+    unless @raw_confirmation_token
+      generate_confirmation_token!
+    end
+
+    opts = pending_reconfirmation? ? { to: unconfirmed_email } : {}
+    if registration_complete
```

Comment:
Could do another ternary here so `send_devise_notification` is called once. Don't but I write this to show I'm paying attention.